src/test/java/com/adyen/RecurringTest.java

```diff
@@ -14,29 +14,25 @@
  *
  * Adyen Java API Library
  *
- * Copyright (c) 2017 Adyen B.V.
+ * Copyright (c) 2020 Adyen B.V.
  * This file is open source and available under the MIT license.
  * See the LICENSE file for more info.
  */
 package com.adyen;
 
-import com.adyen.model.recurring.DisableRequest;
-import com.adyen.model.recurring.DisableResult;
-import com.adyen.model.recurring.RecurringDetail;
-import com.adyen.model.recurring.RecurringDetailsRequest;
-import com.adyen.model.recurring.RecurringDetailsResult;
-import com.adyen.model.recurring.StoreTokenRequest;
-import com.adyen.model.recurring.StoreTokenResult;
+import com.adyen.model.Card;
+import com.adyen.model.recurring.*;
 import com.adyen.service.Recurring;
 import com.adyen.service.exception.ApiException;
 import org.junit.Test;
 
 import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
I think travis will complain about *, can you please revert to the explicit imports?

Reply:
Fixed